packages/sdk-react/src/ui/FileViewer/FileViewer.tsx

```diff
@@ -67,9 +67,26 @@ export const FileViewer = ({
   name,
   onReloadCallback,
 }: FileViewerProps) => {
+  if (!url) return <ErrorComponent onError={onReloadCallback} />;
+
+  if (mimetype === 'application/pdf') return <PdfFileViewer url={url} />;
```

Comment:
I'm wondering, if we should handle a case for mimetypes === 'text/plain' or microsoft office files with application/vnd. ... (docx)  (xlsx) or (pptx)

Reply:
As I know, our API supports images or PDFs only.